katas/jorge-chavez/02-06/ROT 13/ROT13.cs

```diff
@@ -0,0 +1,26 @@
+namespace katas.JorgeChavez;
+
+public class ROT13
+{
+    public static string Rot13(string message)
+    {
+        var alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```

Comment:
Good implementation of the Ceasar cypher, however, how would you implement a more general solution for a Caesar cypher that shifts the letters of a message to any value, for example 7 letters, or 19 letters?